pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -146,34 +133,37 @@ func SetDefaults_KubeControllerManagerConfiguration(obj *KubeControllerManagerCo
 	}
 }
 
-func SetDefaults_GenericComponentConfiguration(obj *GenericComponentConfiguration) {
+func SetDefaults_CloudControllerManagerConfiguration(obj *CloudControllerManagerConfiguration) {
 	zero := metav1.Duration{}
-	if obj.MinResyncPeriod == zero {
-		obj.MinResyncPeriod = metav1.Duration{Duration: 12 * time.Hour}
-	}
-	if obj.ContentType == "" {
-		obj.ContentType = "application/vnd.kubernetes.protobuf"
-	}
-	if obj.KubeAPIQPS == 0 {
-		obj.KubeAPIQPS = 20.0
-	}
-	if obj.KubeAPIBurst == 0 {
-		obj.KubeAPIBurst = 30
+	if obj.ServiceController.ConcurrentServiceSyncs == 0 {
+		obj.ServiceController.ConcurrentServiceSyncs = 1
 	}
-	if obj.ControllerStartInterval == zero {
-		obj.ControllerStartInterval = metav1.Duration{Duration: 0 * time.Second}
+	if obj.NodeStatusUpdateFrequency == zero {
+		obj.NodeStatusUpdateFrequency = metav1.Duration{Duration: 5 * time.Minute}
 	}
 }
 
-func SetDefaults_KubeCloudSharedConfiguration(obj *KubeCloudSharedConfiguration) {
+func SetDefaults_GenericControllerManagerConfiguration(obj *GenericControllerManagerConfiguration) {
 	zero := metav1.Duration{}
-	// Port
 	if obj.Address == "" {
 		obj.Address = "0.0.0.0"
 	}
 	if obj.RouteReconciliationPeriod == zero {
 		obj.RouteReconciliationPeriod = metav1.Duration{Duration: 10 * time.Second}
```

Comment:
Does `RouteReconciliationPeriod` in `KubeCloudSharedConfiguration`?

Reply:
fixed